README.md

```diff
@@ -133,3 +133,7 @@ The values output by this command will need to be inserted manually into astrono
 ## License
 
 The code in this repo is licensed Apache 2.0 with Commons Clause, however it installs Astronomer components that have a commercial license, and requires a commercial subscription from Astronomer, Inc.
+
+## Optional schema validation
+
+The ./bin/values.schema.json file can be used to validate the helm values you are using work with the default airflow chart shipped with this repo. To use them move the file to from bin into the root of this directory and proceed with the helm lint, install, and upgrade commands as normal.
```

Comment:
`bin/values.schema.json` is the wrong place for this file, but I'm not sure what the right place is. Maybe `values.schema.json.example` ? We've abused the `bin` dir in the past and I have been working to reduce that dir to only executables.